src/catnip/runtime/memory/mempool.rs

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+//==============================================================================
+// Imports
+//==============================================================================
+
+use ::dpdk_rs::{
+    rte_mbuf,
+    rte_mempool,
+    rte_pktmbuf_alloc,
+    rte_pktmbuf_clone,
+    rte_pktmbuf_free,
+    rte_pktmbuf_pool_create,
+    rte_socket_id,
+};
+use ::runtime::fail::Fail;
+use ::std::ffi::CString;
+
+//==============================================================================
+// Structures
+//==============================================================================
+
+/// DPDK Memory Pool
+#[derive(Debug)]
+pub struct MemoryPool {
+    /// Underlying memory pool.
```

Comment:
Did you intend to use a [documentation comment](https://doc.rust-lang.org/reference/comments.html) here?

Reply:
Yes. Any suggestion?

Reply:
No, just wondering since we haven't been consistent about using documentation comments throughout the codebase.

Reply:
Every file that I touch I somehow try to provide documentation comments for public API and structures. Eventually we get our documentation in a better shape.